copier.yml

```
@@ -0,0 +1,119 @@
+# CONFIGURATION -------------------------
+_min_copier_version: "9"
+_envops:
+  autoescape: false
+  keep_trailing_newline: true
+_subdirectory: project
+_templates_suffix: .jinja
+_jinja_extensions:
+- copier_templates_extensions.TemplateExtensionLoader
+- extensions.py:CurrentYearExtension
+- extensions.py:GitExtension
+- extensions.py:SlugifyExtension
+_skip_if_exists:
+- CHANGELOG.md
+
+# PROMPT --------------------------------
+project_name:
+  type: str
+  help: Your project name
+
+project_description:
+  type: str
+  help: Your project description
+
+author_fullname:
+  type: str
+  help: Your full name
+  default: "{{ 'Timothée Mazzucotelli' | git_user_name }}"
+
+author_email:
+  type: str
+  help: Your email
+  default: "{{ 'dev@pawamoy.fr' | git_user_email }}"
+
+author_username:
+  type: str
+  help: Your username (e.g. on GitHub)
+  default: pawamoy
+
+repository_provider:
+  type: str
+  help: Your repository provider
+  default: github.com
+  choices:
+  - github.com
+
+repository_namespace:
+  type: str
+  help: Your repository namespace
+  default: "{{ author_username }}"
+
+repository_name:
+  type: str
+  help: Your repository name
+  default: "{{ project_name | slugify }}"
+
+copyright_holder:
+  type: str
+  help: The name of the person/entity holding the copyright
+  default: "{{ author_fullname }}"
+
+copyright_holder_email:
+  type: str
+  help: The email of the person/entity holding the copyright
+  default: "{{ author_email }}"
+
+copyright_date:
+  type: str
+  help: The copyright date
+  default: "{{ current_year }}"
```

```suggestion
  default: "{{ '%Y' | strftime }}"
```

I think we can drop the `CurrentYearExtension` extension with this.
What do you think?

---

Great suggestion, thanks!